yaml/pois.yaml

```diff
@@ -1,16 +1,424 @@
+        carousel, amusement_ride, maze]
+    # no point changing this - it's already masked below any value from
+    # tier5_min_zoom
+    min_zoom: 17
+    output:
+      kind: {col: attraction, tier: 5}
+  # resort
+  - filter: {tourism: resort}
+    min_zoom: { min: [ { max: [ 14, { lit: zoom + 5.32 }, { lit: *tier5_min_zoom } ] }, 17 ] }
+    output: {kind: resort, tier: 5}
+  # theme_park
+  - filter: {tourism: theme_park}
+    min_zoom: { min: [ { max: [ 13, { lit: zoom + 6.32 }, { lit: *tier5_min_zoom } ] }, 17 ] }
+    output: {kind: theme_park, tier: 5}
+  # winery
+  - filter: {tourism: winery}
+    min_zoom: { min: [ { max: [ 14, { lit: zoom + 2.85 }, { lit: *tier5_min_zoom } ] }, 17 ] }
+    output: {kind: winery, tier: 5}
+
+  ############################################################
+  # TIER 6
+  ############################################################
+  # allotments - no POI
+  # artwork, hanami
+  - filter:
+      tourism: [artwork, hanami, trail_riding_station]
+    # already clamped below polygon area min
+    min_zoom: 17
+    output:
+      kind: {col: tourism, tier: 6}
+  # common - no POI
+  # garden
+  - filter: {leisure: garden}
+    min_zoom: { min: [ 12, { max: [ { lit: zoom }, { lit: *tier6_min_zoom } ] }, 16 ] }
+    output: {kind: garden, tier: 6}
+  # hedge - no POI
+  # pedestrian - no POI
+  # pitch
+  - filter: {leisure: pitch}
+    # already clamped below area min
+    min_zoom: 16
+    output:
+      kind: pitch
+      kind_detail: {col: sport}
+      tier: 6
+  # place_of_worship
+  - filter: {amenity: place_of_worship}
+    min_zoom: { min: [ { max: [ { lit: 2 * zoom - 9.55 }, { lit: *tier6_min_zoom } ] }, 17 ] }
+    output: {kind: place_of_worship, tier: 6 }
+  # playground
+  - filter: { leisure: playground }
+    # already clamped below area limit
+    min_zoom: 17
+    output:
+      kind: {col: leisure, tier: 6}
+  # school
+  - filter: {amenity: school}
+    min_zoom: { min: [ { max: [ { lit: zoom + 2.3 }, { lit: *tier6_min_zoom } ] }, 15 ] }
+    output: {kind: school, tier: 6}
+  # tree_row - no POI
+  # wilderness_hut
+  - filter: {tourism: wilderness_hut}
+    min_zoom: 15
+    output: {kind: wilderness_hut, tier: 6}
+
+  ############################################################
+  # TIER 6 EXTRA - PARKING
+  ############################################################
+  # parking
+  - filter: {amenity: parking}
+    min_zoom: 17
+    output: {kind: parking, tier: 6}
+
+  ############################################################
+  # NOT IN ANY TIER
+  ############################################################
+  - filter: {historic: battlefield, operator: *us_forest_service}
+    min_zoom: GREATEST(10, LEAST(zoom + 4.0, 17))
+    output: {kind: battlefield}
+  # park w/ US Parks Service operator
+  - filter:
+      any:
+        leisure: park
+        landuse: park
+    min_zoom: GREATEST( LEAST(zoom + 2, 14), 9)
```

Comment:
The original logic excluded parks with the US Parks Service operator from the tier, so this is the "else" clause to ensure that things not caught by previous expressions still get assigned a `kind`, if not a `tier`.